testbed/core/urls/api_urls.py

```diff
@@ -28,6 +31,24 @@
         followers_collection,
         name="followers-collection",
     ),
+    # LOLA Content Collection: Raw authored objects, LOLA authentication required
+    path(
```

Comment:
All the repetition of "actors/<int:pk" can be done once: see https://docs.djangoproject.com/en/5.2/topics/http/urls/ where it says "We can improve this by stating the common path prefix only once and grouping the suffixes that differ"